tutorial/02-materials.ipynb

```diff
@@ -576,10 +576,10 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "It is often also useful to be able to convert the current mass-weighted material to \n",
+      "It is also useful to be able to convert the current mass-weighted material to \n",
```

Comment:
No need to remove `often`, no need to keep it though either, I guess. Non-blocking.